README.md

```diff
@@ -74,7 +74,7 @@ bower install
 # Create a .env file and populate it with the necessary API keys and secrets:
 touch .env
 ```
-
+(Note if you're having issues with any of these steps or have installed something with npm delete your modules AND clean your npm cache with npm cache clear)
```

Comment:
> (Note if you're having issues with any of these steps or have installed something with npm **COMMA HERE** delete your modules AND clean your npm cache with `npm cache clear`)

Also added back ticks around "`npm cache clear`"

Also, I'm not really sure what you mean by "installed something with npm".